packages/frontend/src/components/ProjectAccess/ProjectAccess.tsx

```diff
@@ -213,8 +244,13 @@ const ProjectAccess: FC<ProjectAccessProps> = ({ projectUuid }) => {
         }),
     );
 
-    if (isProjectAccessLoading || isOrganizationUsersLoading) {
-        return <NonIdealState title="Loading..." icon={<Spinner />} />;
+    if (isProjectAccessLoading || isProjectAccessLoading) {
```

Comment:
this removes the ability to check if `isOrganizationUsersLoading` is true. Can we bring it back?